docs/moduleAna1Tests.rst

```diff
@@ -160,3 +160,173 @@ The simulation results are plotted alongside the analytical solution for the giv
+conservation equation:
+
+.. math::
+    \bar{z}_0 = \bar{z}_{t} + \frac{\overline{v^2}_{t}}{2g}
+    + tan\alpha \bar{s}_t
+   :label: altitude-energy-conservation-avg
+
+where the mass average :math:`\bar{a}` value of a quantity :math:`a` is:
+
+.. math::
+    \bar{a} = \frac{\sum_k m_k a_k}{\sum_k m_k}
+
+This means that the mass averaged quantities also follow the same energy
+conservation law when express in terms of altitude. The same figure as in
+:numref:`fig-altitude-energy-conservation` can be drawn for the center of mass
+profile path.
+
+The aim is now to take advantage of this energy conservation line to evaluate
+the DFA simulation. If we compute for the particles in the simulation, the
+mass averaged path profile and compare it to the :math:`\alpha` line,
+we will be able to compute the error compared to the run-out predicted
+by the energy line as well as the error on the velocity altitude. The following
+figures illustrate the problem.
+
+.. list-table::
+
+    * -
+
+        .. figure:: _static/energyLinePath.png
+            :width: 90%
+
+            View of the avalanche simulation and extracted path
+
+
+      -
+
+        .. figure:: _static/energyLine.png
+            :width: 90%
+
+            Simulation path profile with run-out line, :math:`\alpha`
+            line and energy points
+
+From the different mass averaged simulation quantities and the theoretical
+:math:`\alpha` line it is possible to extract four error indicators.
+The three first ones are related to the run-out. The intersection between
+the :math:`\alpha` line and the mass averaged path profile (or its extrapolation
+if the profile is too short) defines the theoretical run-out point.
+The horizontal and vertical distance between this point and the end of the path
+profile defines a run-out error in meters. The angle difference between
+the :math:`\alpha` line angle and the DFA simulation run-out line defines
+the run-out angle error.
+Finally, the Root Mean Square Error (RMSE) between the :math:`\alpha` line
+and the DFA simulation energy points defines an error on the velocity altitude
+:math:`\frac{\overline{v^2}}{2g}`.
+
+
+Limitations and remarks
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+We now have to pay attention to stay in the good range of application of
+this test. Indeed, one of the important hypothesis when developing
+the energy solution, is that the material is flowing in the steepest
+slope direction. If this hypothesis fails, then it is not possible to develop
+the analytic energy solution anymore. In the 3D case, the distance vector
+:math:`\mathbf{dl}` traveled by the particles reads
+:math:`dl = \frac{ds}{cos(\gamma)}`, where :math:`\gamma` is the angle
+between the :math:`\mathbf{dl}` vector and the horizontal plane
+which can differ from the slope angle :math:`\theta`. If this happens
+it is not possible to differentiate anymore the DFA simulation error from
+the hypothesis fail error. The test has then to be used more qualitatively
```

Comment:
I do not get the sentence? (even trying french sentence structure does not help in this case :rofl: )

Reply:
What I want to say here is that in 3D, we have an error coming from the fact that one of the fundamental hypothesis for the energy solution is not satisfied so the runout error will be a sum of the error coming from the unsatisfied hypothesis and the error coming from our numerics (too big dt, to few particles or bad implementation)